src/datamodule.py

```diff
@@ -80,6 +81,9 @@ def __getitem__(self, idx):
 
         # remove nans and convert to tensor
         cube["pixels"] = torch.nan_to_num(torch.as_tensor(data=cube["pixels"]), nan=0.0)
+        cube["bbox"] = torch.as_tensor(data=cube["bbox"], dtype=torch.float64)
+        cube["epsg"] = torch.as_tensor(data=cube["epsg"], dtype=torch.int32)
+        cube["date"] = str(cube["date"])
         cube["latlon"] = torch.as_tensor(data=cube["latlon"])
         cube["timestep"] = torch.as_tensor(data=cube["timestep"])
         cube["path"] = str(chip_path.absolute())
```

Comment:
Still need this rename `cube["path"]` in ClayDataModule to match with GeoTIFFDataModule's `cube["source_url"]`, but will do that in a follow-up PR when I handle listing of GeoTIFF files from s3 buckets.